README.md

```diff
@@ -13,6 +13,14 @@ You can find the deployed project at [https://www.expresswash.us/](https://www.e
 |                  [<img src="https://github.com/favicon.ico" width="15">](https://github.com/MattGThomas)                  |                          [<img src="https://github.com/favicon.ico" width="15">](https://github.com/quan005)                           |                        [<img src="https://github.com/favicon.ico" width="15">](https://github.com/RyanAnsberry)                        |                  [<img src="https://github.com/favicon.ico" width="15">](https://github.com/malarc01)                   |                   [<img src="https://github.com/favicon.ico" width="15">](https://github.com/cahaug)                    |                  [<img src="https://github.com/favicon.ico" width="15">](https://github.com/sam-malanchuk)                   |
 |        [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/)        | [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/in/daquan-freeman-a4b63622/) | [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/in/ryan-ansberry-11a55415a/) | [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/in/malarc01/) | [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/in/calehaug/) | [<img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15">](https://www.linkedin.com/in/sam-malanchuk/) |
 
```

Comment:
Thanks for updating the team, the readme still needs the information below this updated.